README.md

```diff
@@ -8,6 +8,33 @@ This is an educational project to make a To Do List web application without usin
 
 - HTML5
 - JavaScript
+- npm
+- webpack
+
+## Screenshot
+
+![screenshot](./screenshot.png)
+
+## Live Demo
+
+Not available yet
+
```

Comment:
- [ ] (OPTIONAL) Since a live demo link is not available, it would be better to remove this section as it now serves no purpose in order to make the README file more professional 💯

Reply:
done